test/distributed/pipelining/test_schedule.py

```diff
@@ -10,8 +10,15 @@
     ScheduleLoopedBFS,
 )
 from torch.distributed.pipelining.schedules import (
+    _Action,
+    _add_unshard_reshard,
     _format_pipeline_order,
     _validate_pipeline_order,
+    B,
+    F,
+    RESHARD,
+    UNSHARD,
+    W,
```

Comment:
nit: can you make B, F, W more descriptive? Can be in a separate PR if it would involve larger changes. Thanks

Reply:
Wdym, name them longer names?  I wanted them to be 1 character so they were easy to read/type in the instantiations below.  I'm happy to add more comments but hopefully you don't insist on writing it out as 'FORWARD' everywhere?